docs/docsite/rst/user_guide/playbooks_conditionals.rst

```diff
@@ -1,60 +1,144 @@
 .. _playbooks_conditionals:
 
+************
 Conditionals
-============
+************
 
-.. contents:: Topics
+In a playbook, you may want to execute different tasks, or have different goals, depending on the value of a fact (data about the remote system), a variable, or the result of a previous task. You may want the value of some variables to depend on the value of other variables. Or you may want to create additional groups of hosts based on whether the hosts match other criteria. You can do all of these things with conditionals.
 
-
-Often the result of a play may depend on the value of a variable, fact (something learned about the remote system), or previous task result.
-In some cases, the values of variables may depend on other variables.
-Additional groups can be created to manage hosts based on whether the hosts match other criteria. This topic covers how conditionals are used in playbooks.
+Ansible uses Jinja2 :ref:`tests <playbooks_tests>` and :ref:`filters <playbooks_filters>` in conditionals. Ansible supports all the standard tests and filters, and adds some unique ones as well.
 
 .. note::
 
-  There are many options to control execution flow in Ansible. More examples of supported conditionals can be located here: https://jinja.palletsprojects.com/en/master/templates/#comparisons.
+  There are many options to control execution flow in Ansible. You can find more examples of supported conditionals at `<https://jinja.palletsprojects.com/en/master/templates/#comparisons>`_.
 
+.. contents::
+   :local:
 
 .. _the_when_statement:
 
-The When Statement
-``````````````````
+Basic conditionals with ``when``
+================================
+
+The simplest conditional statement applies to a single task. Create the task, then add a ``when`` statement that applies a test. The ``when`` clause is a raw Jinja2 expression without double curly braces (see :ref:`group_by_module`). When you run the task or playbook, Ansible evaluates the test for all hosts. On any host where the test passes (returns a value of True), Ansible runs that task. For example, if you are installing mysql on multiple machines, some of which have SELinux enabled, you might have a task to configure SELinux to allow mysql to run. You would only want that task to run on machines that have SELinux enabled:
+
+.. code-block:: yaml
+
+    tasks:
+      - name: Configure SELinux to start mysql on any port
+        seboolean: name=mysql_connect_any state=true persistent=yes
+        when: ansible_selinux.status == "enabled"
+        # all variables can be used directly in conditionals without double curly braces
```

Comment:
Is this using a variable or a fact?  ... asking for a friend :-)

Reply:
I believe `ansible_selinux.status` is a fact that has been injected into a "top-level" variable so you can access it without the full `ansible_fact` syntax